src/ResourceManager/OperationalInsights/ChangeLog.md

```diff
@@ -23,6 +23,7 @@
 * Added Location Completer to -Location parameters allowing tab completion through valid Locations
 * Added ResourceGroup Completer to -ResourceGroup parameters allowing tab completion through resource groups in current subscription
 * Obsoleted -Tags in favor of -Tag for New-AzureRmOperationalInsightsSavedSearch, Set-AzureRmOperationalInsightsSavedSearch, New-AzureRmOperationalInsightsWorkspace, and Set-AzureRmOperationalInsightsWorkspace
+* Added support for V2 API querying via Invoke-AzureRmOperationalInsightsQuery. See [https://dev.loganalytics.io/](https://dev.loganalytics.io/) for more info on the new API.
```

Comment:
@csuich2 @alexeldeib please move this line you added to be under the `## Current Release` header -- merging in the latest changes from the `preview` branch moved this under the latest release's notes